docs/benchmarks/benchmark.rst

```diff
@@ -151,17 +151,17 @@ file.
 
    `Driving SMARTS V0` result:
    - completion: 1
-   - humanness: 0.2
-   - rules: 0.8
-   - time: 0.3
-   - overall: 1.7
+   - humanness: 0.7
+   - rules: 0.9
+   - time: 0.2
+   - overall: 0.504
 
 A particular version of a benchmark can be targeted using a modified
 syntax ``benchmark_name==version``:
 
 .. code:: bash
 
-   > scl benchmark run driving_smarts==0.0 ...
+   $ scl benchmark run driving_smarts==0.0 "./baselines/driving_smarts/v0/agent_config.yaml" --auto-install
```

Comment:
The option is not necessary here I think.

```suggestion
   $ scl benchmark run driving_smarts==0.0 "./baselines/driving_smarts/v0/agent_config.yaml"
```

Reply:
Changed in PR #1844 .